Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -699,6 +699,8 @@ search=Suche
 start=Start
 startDate=Startdatum
 startIndexing=Indexierung starten
+startIndexingAll=Indexierung starten f\u00FCr allen
+startIndexingRemaining=Indexierung starten f\u00FCr nicht indexiert
```

Comment:
A maybe better translation would be `Indexierung f\u00FCr nicht Indexierte starten`

Reply:
Hmm… tough one… maybe `Indexierung vervollständigen`?